testing/integration.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
-set -e
```

Comment:
This is because, if we have a failing test, then the script would stop immediately at

```bash $module/harness/harness.sh```

Instead, we want the script to continue, since we have the mechanism to handle errors directly in the code.